code/modules/1713/lift/pseudoturf.dm

```diff
@@ -0,0 +1,108 @@
+// this is a lift pseudoturf, an object that takes the appearance and layer
```

Comment:
Pseudoturfs wont work without the other 3000000000 files that need to be changed